index.d.ts

```diff
@@ -60,6 +60,8 @@ interface EggMySQL {
     where?: object,
     condition?: EggMySQLCondition
   ): Promise<EggMySQLGetResult>;
+  /// get counts from table 
+  count: (table: string, values: object) => Promise<number>;
```

Comment:
values should name to where, keep the same as the real count method https://github.com/ali-sdk/ali-rds/blob/master/lib/operator.js#L62

```js
count: (table: string, where?: object)
```

Reply:
done